_data/projects/web-dev.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: WebDev
+desc: simple but cool webpages made using css flexbox and html
+site: https://github.com/RasenRhino/WebDev.git
+tags:
+- css
+- flexbox
+- html
+- interactive
+
+upforgrabs:
+  name: make it better!!
+  link: https://github.com/RasenRhino/WebDev.git
```

Comment:
This link should point to a curated list of issues that new contributors can pick up.

I had a look at the repository but I couldn't spot any issues. Is that something you're interested in adding?